pytorch_forecasting/metrics/base_metrics.py

```diff
@@ -87,15 +87,30 @@ def rescale_parameters(
         """  # noqa: E501
         return encoder(dict(prediction=parameters, target_scale=target_scale))
 
-    def to_prediction(self, y_pred: torch.Tensor) -> torch.Tensor:
-        """
-        Convert network prediction into a point prediction.
+    def _to_prediction_3d(self, y_pred: torch.Tensor) -> torch.Tensor:
+        """Convert network prediction into a point prediction.
 
-        Args:
-            y_pred: prediction output of network
+        This is an internal helper method.
 
-        Returns:
-            torch.Tensor: point prediction
+        Parameters
```

Comment:
good docstring!

Regarding the API: is this "as current" or "as reworked"?

* how do we ensure downwards compatibility with v1
* did we not decide on `(batch, variable, time, params)`?

This is as current, right?